package.json

```diff
@@ -72,8 +72,8 @@
     "build:esm": "rollup -c",
     "build:esm:mock-classnames": "mock_classnames=on rollup -c",
     "build:react-icons": "svg2react-icon --typescript --keep-colors node_modules/monday-ui-style/src/Icons src/components/Icon/Icons",
-    "storybook": "storybook=true start-storybook -p 7008",
-    "build-storybook": "npm run build:storybook-exports && storybook=true build-storybook",
+    "storybook": "storybook=true storybook dev -p 7008",
```

Comment:
I saw you removed references that check 'storybook' env var in the setup of storybook (where you check IS_DEV), so is there's still need for storybook=true env variable?